common/config/rush/command-line.json

```diff
@@ -148,7 +148,14 @@
       "enableParallelism": true,
       "allowWarningsInSuccessfulBuild": true
     },
-
+    {
+      "commandKind": "bulk",
+      "name": "execute:samples",
+      "summary": "Execute samples defined in projects",
+      "description": "Execute samples against external service.",
```

Comment:
Why is it relevant to say that sample is running against an external service? Samples always run against real, live external service.

I would suggest `Execute samples using the source code`

Reply:
I wouldn't have necessarily known that if it hadn't been called out here. So I think there's value in both. Maybe `Execute project samples against live service instances`. In the description it would also be nice to know whether I am expected to create my own service instances first (and how to provide endpoints / creds), or whether the samples will create/destroy them for me (in which case, what configuration do I need to provide and how?)

Reply:
As far as I understood, the intention to have the script is to ensure that samples are not going stale while changes are happening in the source code. Whether they run against the live service or not is irrelevant to this goal. If we have emulators in the future to mimic live services, the samples can be free to use them and this script shouldnt care.

Ideally this step would include having a step that changes the published package reference with source code. Storage samples point to source code already, but it should ideally point to the published package `@azure/storage-blob`, so that people who copy paste the sample have the least amount of work to do to run the sample. This can be done in a separate PR.

Regarding endpoints and creds, the samples should be using the same endpoints and creds as the nightly tests. So the onus is on the sample writer to use the right environment variables.

Reply:
> In the description it would also be nice to know whether I am expected to create my own service instances first (and how to provide endpoints / creds), or whether the samples will create/destroy them for me (in which case, what configuration do I need to provide and how?)

I think @bsiegel is probably referring to how the customers will be running the samples

Reply:
Updated the description to `Execute samples using the source code`